common/app/views/mainAMP.scala.html

```diff
@@ -19,6 +20,9 @@
         <script custom-element="amp-youtube" src="https://cdn.ampproject.org/v0/amp-youtube-0.1.js" async></script>
         <script custom-element="amp-instagram" src="https://cdn.ampproject.org/v0/amp-instagram-0.1.js" async></script>
         <script custom-element="amp-list" src="https://cdn.ampproject.org/v0/amp-list-0.1.js" async ></script>
+        @if(LiveUpdateAmpSwitch.isSwitchedOn) {
```

Comment:
&& is a live blog ? or doesn't have a negative impact?  as the js increases we should try to be selective what we include if it would slow things down